### PR TITLE
Using new class to serialize data for json download

### PR DESCRIPTION
**Pull request by carlosribas**

### CODE REVIEW

File: rnacentral/export/views.py [hunk 1]
```diff
@@ -73,24 +72,25 @@ def format_output(rnacentral_ids):
...
             serializer_context = {'request': fake_request}
-            serializer = RnaFlatSerializer(queryset, context=serializer_context, many=True)
+            serializer = RnaPrecomputedJsonSerializer(queryset, context=serializer_context, many=True)
 
             renderer = renderers.JSONRenderer()
             output = renderer.render(serializer.data)
             # omit opening and closing square brackets for easy concatenation
             output = output[1:-1]
             # make relative urls absolute
-            output = output.replace('"/api/v1/', '"https://rnacentral.org/api/v1/')
```
Comment (AntonPetrov) [inline rnacentral/export/views.py]: Does this need to be commented out now?

File: rnacentral/apiv1/serializers.py [hunk 2]
```diff
@@ -546,3 +546,14 @@ class EnsemblComparaSerializer(serializers.ModelSerializer):
...
+
+
+class RnaPrecomputedJsonSerializer(serializers.ModelSerializer):
+    """Serializer class for the Json download."""
+    sequence = serializers.CharField(source='get_sequence', read_only=True)
+
+    class Meta:
+        model = RnaPrecomputed
+        fields = (
+            'id', 'rna_type', 'description', 'databases', 'sequence'
```
Comment (AntonPetrov) [inline rnacentral/apiv1/serializers.py]: Currently `databases` is a comma separated list - would it make sense to make it an array in json?

File: rnacentral/export/views.py (hunk 1 above)
Comment (carlosribas) [reply rnacentral/export/views.py]: Well, if I'm not wrong, this line is used to show the correct URL. We are not showing urls in this new json file, at least not now.

File: rnacentral/apiv1/serializers.py (hunk 2 above)
Comment (carlosribas) [reply rnacentral/apiv1/serializers.py]: We are showing as a string, like this: "databases":"ENA,Ensembl,Rfam"

Comment (AntonPetrov) [reply rnacentral/apiv1/serializers.py]: that's right, good point

File: rnacentral/export/views.py (hunk 1 above)
Comment (AntonPetrov) [reply rnacentral/export/views.py]: that's right, good point

File: rnacentral/apiv1/serializers.py (hunk 2 above)
Comment (AntonPetrov) [reply rnacentral/apiv1/serializers.py]: yes, I was thinking maybe split it like this: "databases": ["ENA", "Ensembl", "Rfam"]?

Comment (carlosribas) [reply rnacentral/apiv1/serializers.py]: Okay, I'll check it...

Comment (carlosribas) [reply rnacentral/apiv1/serializers.py]: Done

**Pull request merged**